docs/developers/tutorial.md

```diff
@@ -742,13 +742,13 @@ argument is not required.
 
 ### Adding custom resolver
 
-Update the `scalafixResolvers` setting to resolve rules that are published to a
+Update the `scalafixCustomResolvers` setting to resolve rules that are published to a
 custom repository like Bintray or a private Nexus.
 
 ```scala
 // build.sbt
 import com.geirsson.coursiersmall.{Repository => R}
-scalafixResolvers.in(ThisBuild) ++= List(
+scalafixCustomResolvers.in(ThisBuild) ++= List(
```

Comment:
Where is this setting defined? I can't see it in `ScalafixPlugin` https://github.com/scalacenter/sbt-scalafix/blob/master/src/main/scala/scalafix/sbt/ScalafixPlugin.scala#L22

It's not available from sbt shell

```
sbt:default-9e2deb> scalafixCustomResolvers
[error] Not a valid command: scalafixCustomResolvers
[error] Not a valid project ID: scalafixCustomResolvers
[error] Expected ':'
[error] Not a valid key: scalafixCustomResolvers (similar: scalafixResolvers, fullResolvers)
[error] scalafixCustomResolvers
[error]                        ^
```